db/models.py

```diff
@@ -1,10 +1,11 @@
 from django.db import models
+from django.db.models import CharField
 
 
 class Genre(models.Model):
     name = models.CharField(max_length=255, unique=True)
 
-    def __str__(self) -> str:
+    def __str__(self) -> CharField:
```

Comment:
str is enough
also why to not use models.CharField but import it separately? Because otherwise you should follow the same approach everywhere and use CharField instead of models.CharField that isn't actually good